Neos.Flow/Classes/Package.php

```diff
@@ -130,7 +132,7 @@ public function boot(Core\Bootstrap $bootstrap)
         $dispatcher->connect(Tests\FunctionalTestCase::class, 'functionalTestTearDown', Mvc\Routing\RouterCachingService::class, 'flushCaches');
 
         $dispatcher->connect(Configuration\ConfigurationManager::class, 'configurationManagerReady', function (Configuration\ConfigurationManager $configurationManager) {
-            $configurationManager->registerConfigurationType('Views', Configuration\ConfigurationManager::CONFIGURATION_PROCESSING_TYPE_APPEND);
+            $configurationManager->registerConfigurationType('Views', new AppendConfigurationSource(new YamlSource(), 'Views'));
```

Comment:
In this special case you mentioned, it is not necessary. The `registerConfigurationType` converts the given CONFIGURATION_PROCESSING_TYPE_ to the corresponding loader.

But the third parameter `$allowSplitSource` is removed because it is hardcoded in the implemented loader's - like it is currently done by [`$configurationTypes`](https://github.com/neos/flow-development-collection/blob/5735c345d356bc8664e0ef43d91570d44e1a3c14/Neos.Flow/Classes/Configuration/ConfigurationManager.php#L119).  
So, yes there is a breaking change.